account/routes.js

```diff
@@ -62,7 +61,7 @@ router.get('/sign-in/success', requiresAuth(), async (req, res, next) => {
 
 router.get('/sign-in', (req, res) =>
     res.oidc.login({
-        returnTo: '/account/sign-in/success',
+        returnTo: getValidReferrerOrDefault(req?.session?.referrer),
```

Comment:
I don't think this is right, it will work for the dashboard but break elsewhere? The valid referrer util doesn't return `/account/sign-in/success` it returns an apply route.